language/move-lang/functional-tests/tests/diem/on_chain_config/change_script_allowlist.move

```diff
@@ -123,3 +90,51 @@ module FooConfig2 {
     }
 }
 // check: INVALID_MODULE_PUBLISHER
+
+// disable all non-admin txes
+//! new-transaction
+//! sender: diemroot
+script {
+use 0x1::DiemAccount;
+fun main(account: &signer) {
+    DiemAccount::halt_all_transactions(account);
+}
+}
+// check: "Keep(EXECUTED)"
+
+// sending fails
+//! new-transaction
+script {
+fun main() {
+}
+}
+// check: "Discard(UNKNOWN_SCRIPT)"
+
+// re-enable
+//! new-transaction
+//! sender: diemroot
+script {
+use 0x1::DiemAccount;
+fun main(account: &signer) {
+    DiemAccount::resume_transactions(account);
+}
+}
+// check: "Keep(EXECUTED)"
+
+// sending succeeds again
+//! new-transaction
+script {
+fun main() {
+}
+}
+// check: "Discard(UNKNOWN_SCRIPT)"
```

Comment:
This comment and the check is confusing since it's the same as the failure case. Could you possibly clarify in the comment why this is a success and the test above is a failure?